fix-project-terminating/README.md

```diff
@@ -2,6 +2,47 @@
 
 In some cases, projects/namespaces get stuck in a `terminating` state. In most cases this is due to one or more finalizers on objects within the projects  - including for Custom Resources (CRs) based on Custom Resource Definitions (CRDs). For these, the following `patch` command can be used to remove the finalizer and hence recover from the terminating state.
 
+### Find out what's holding up the termination of the project
+
+```bash
+> oc get project <project-name> -o yaml
+```
+
+Look in the `status` of the output to see what needs to be patched. In the example below we'll need to patach `cephclusters.ceph.rook.io`
+
+```
+...
+status:
+  conditions:
+  - lastTransitionTime: "2020-03-26T10:01:05Z"
```

Comment:
Do we feel this section is all needed, or do we rather just cut it down to the one that shows the message of interest? I.e.: the "Some content in the namespace ..."

Reply:
Sure, updated